Reqnroll/Infrastructure/TestExecutionEngine.cs

```diff
@@ -124,6 +115,20 @@ public virtual async Task OnTestRunStartAsync()
             await FireEventsAsync(HookType.BeforeTestRun);
         }
 
+        async Task TryTransmitReqnrollProjectRunningEventAsync()
+        {
+            try
+            {
+                var testAssemblyName = _testRunnerManager.TestAssembly.GetName().Name;
+                var projectRunningEvent = _analyticsEventProvider.CreateProjectRunningEvent(testAssemblyName);
+                await _analyticsTransmitter.TransmitReqnrollProjectRunningEventAsync(projectRunningEvent);
+            }
+            catch (Exception)
+            {
+                // catch all exceptions since we do not want to break anything
```

Comment:
Can we add something to a log to at least get visibility of an error? Stopping the exception is the right call, but dropping it on the floor seems wrong.

Reply:
I don't think there is any log that is accessible at that point and we could use.

I usually add a `Debug.WriteLine` for such cases, so that we can at least see the problems during debugging with [DebugView](https://learn.microsoft.com/en-us/sysinternals/downloads/debugview).